### ISSUE TITLE
How to configure plugin to fully generate deferred code?

### ISSUE THREAD

**Issue by slavap** [localizely/flutter-intl-vscode#44]
Currently generated code for deferred looks like:
```dart
import 'messages_en.dart' as messages_en;
import 'messages_es.dart' as messages_es;
import 'messages_nb.dart' as messages_nb;

typedef Future<dynamic> LibraryLoader();
Map<String, LibraryLoader> _deferredLibraries = {
  'en': () => new Future.value(null),
  'es': () => new Future.value(null),
  'nb': () => new Future.value(null),
};
```

But it should be generated differently to support deferred on Flutter Web:
```dart
import 'messages_en.dart' deferred as messages_en;
import 'messages_es.dart' deferred as messages_es;
import 'messages_nb.dart' deferred as messages_nb;

typedef Future<dynamic> LibraryLoader();
Map<String, LibraryLoader> _deferredLibraries = {
  'en': () => messages_en.loadLibrary(),
  'es': () => messages_es.loadLibrary(),
  'nb': () => messages_nb.loadLibrary(),
};
```

How to configure plugin to do proper code generation ? Or it is just a bug?

**Comment by lzoran** [comment]
Hi @slavap,

Currently, deferred is not supported. 
We plan to add such support later, depending on demand from users.

**Comment by slavap** [comment]
@lzoran Then consider this issue as demand ;-) Actually minor changes in existing codegen are needed. It works well after my manual changes already.

**Comment by aleksakrstic** [comment]
Hi @slavap 
It is available with IDE extension `1.13.0`:

    flutter_intl:
      ...
      use_deferred_loading: false # Optional. Must be set to true to generate localization code that is loaded with deferred loading. Default: false

**Comment by slavap** [comment]
@aleksakrstic Thank you! It works.
